src/js/components/ContainerSettingsComponent.jsx

```diff
@@ -273,7 +274,10 @@ var ContainerSettingsComponent = React.createClass({
         <div className="duplicable-list">{this.getPortMappingRows()}</div>
         <h4>Parameters</h4>
         <div className="duplicable-list">{this.getParametersRows()}</div>
-        <div>You can set your Docker volume settings below.</div>
+        <div>
+          You can set your Docker volume
+          settings <u onClick={() => this.props.openVolumes(true)}>below</u>.
```

Comment:
Why are we using a `u` element rather than an `a` here? Do we not want it to gain focus?